conda-envs/mlos_core-3.11.yml

```diff
@@ -28,5 +28,6 @@ dependencies:
     - bump2version
     - licenseheaders
     - pytest-local-badge
+    - sqlalchemy
```

Comment:
Suggestion to put this into the `mlos_bench/setup.py` as an `extra_requires` and it should get handled automatically in all of the conda envs then.
(`full` specifier below gets mapped to all different `extra_requires` combos for convenience).

Reply:
done!